docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -363,6 +363,10 @@ children:
           url: Validation-ORM-layer.html
           output: 'web, pdf'
 
+    - title: 'Application Configuration'
```

Comment:
Maybe `Configuring Applications` or `Customizing Applications` is a better title to be consistent with others that use `...ing`

Reply:
Noted. Thanks